src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import './variables';
+
 body {
   margin: 0;
+  height: 100vh;
```

Comment:
It's not recommended to use viewport units for height. This can cause issues on mobile devices when the address bar appears/disappears. Consider using a different unit, like `em` or `rem`.